sdk/search/search/src/serviceModels.ts

```diff
@@ -55,9 +55,47 @@ import {
   TagScoringFunction,
   TextWeights,
   ScoringFunctionAggregation,
-  RegexFlags
+  RegexFlags,
+  ConditionalSkill,
+  KeyPhraseExtractionSkill,
+  OcrSkill,
+  ImageAnalysisSkill,
+  LanguageDetectionSkill,
+  ShaperSkill,
+  MergeSkill,
+  EntityRecognitionSkill,
+  SentimentSkill,
+  SplitSkill,
+  TextTranslationSkill,
+  WebApiSkill,
+  DefaultCognitiveServicesAccount,
+  CognitiveServicesAccountKey
 } from "./generated/service/models";
 
+/**
+ * Options for a list skillsets operation.
+ */
+export interface ListSkillsetsOptions extends OperationOptions {
```

Comment:
this interface and the one below seem wrong. I don't think they take a select clause?

There don't appear to be any parameters here: https://docs.microsoft.com/en-us/rest/api/searchservice/list-skillset

Maybe the swagger is wrong?

Reply:
List Skillset does not have select. Only List Index & List Synonymmap have select. My bad. removed it.